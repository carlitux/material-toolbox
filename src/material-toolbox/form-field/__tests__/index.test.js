// @flow
import React from 'react';
import { shallow } from 'enzyme';

import FormField from '../index';

describe('component::Ripple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(
      <FormField label="Should render label">
        <input type="button" />
      </FormField>,
    );
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot cssOnly', () => {
    const component = shallow(
      <FormField label="Should render label" cssOnly>
        <input type="button" />
      </FormField>,
    );
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot alignEnd', () => {
    const component = shallow(
      <FormField label="Should render label" alignEnd>
        <input type="button" />
      </FormField>,
    );
    expect(component).toMatchSnapshot();
  });
});
