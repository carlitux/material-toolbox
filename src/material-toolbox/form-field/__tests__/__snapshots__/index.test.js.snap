// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::Ripple Should match snapshoot 1`] = `
<div
  className="mdc-form-field"
>
  <input
    key=".0"
    type="button"
  />
  <label
    onClick={[Function]}
  >
    Should render label
  </label>
</div>
`;

exports[`component::Ripple Should match snapshoot alignEnd 1`] = `
<div
  className="mdc-form-field mdc-form-field--align-end"
>
  <input
    key=".0"
    type="button"
  />
  <label
    onClick={[Function]}
  >
    Should render label
  </label>
</div>
`;

exports[`component::Ripple Should match snapshoot cssOnly 1`] = `
<div
  className="mdc-form-field"
>
  <input
    key=".0"
    type="button"
  />
  <label
    onClick={[Function]}
  >
    Should render label
  </label>
</div>
`;
